tests/ConfigTest.php

```diff
@@ -48,13 +45,13 @@ public function testCustomParams(): void
                 ],
             ],
         ];
-        $params['yiisoft/form']['defaultConfig'] = 'simple';
+        $params['yiisoft/form']['defaultThemeName'] = 'simple';
 
         $bootstrapList = $this->getBootstrapList($params);
         $bootstrap = array_shift($bootstrapList);
         $bootstrap(new SimpleContainer());
 
-        $input = Field::hidden(new HiddenForm(), 'key');
+        $input = Hidden::widget()->formAttribute(new HiddenForm(), 'key');
```

Comment:
That is significant regression is readability.

Reply:
In master `Field` merge package-specific configurations (common configuration in field factory, individual configuration in field factory, configuration at the place of use) and pass it to widget factory from `yiisoft/widget`.

In this PR use new feature of `yiisoft/widget` (PR)In master `Field` merge package-specific configurations (common configuration in field factory, individual configuration in field factory, configuration at the place of use) and pass to `WidgetFactory` from `yiisoft/widget`.

This PR use new feature of `yiisoft/widget` (see https://github.com/yiisoft/widget/pull/84). Now package-specific configuration merged into widget, `Field` not need for it.

But we can create new `Field` that will be used only as shortcutrs. How is it done in Yii HTML (helper `HTML`).

Reply:
Here I'm talking only about how it looks from user perspective. HTML and widgets are used a lot and developer types their code and reads this code constantly so the following is a significant difference:

```php
Field::hidden(new HiddenForm(), 'key');

// vs

Hidden::widget()->formAttribute(new HiddenForm(), 'key');
```

Reply:
Yes, I agree with it.

Reply:
Goal of PR make `Field::hidden(new HiddenForm(), 'key');` === `Hidden::widget()->formAttribute(new HiddenForm(), 'key');`.

In master `Field::...` use package-specific configuration, but `Hidden::...` don't use. In PR both calls returns same result.

Reply:
Then why do we change all tests and phpdoc to use `widget()`?

Reply:
> Then why do we change all tests and phpdoc to use `widget()`?

Not need. I'm added `Field` with static shortcut methods.